docs(pie-docs): content added and amended for the Engineers section of the docs site (#298)
justeattakeaway/pie: apps/pie-docs/src/engineers/getting-started/structure.md

    order: 2
---

## PIE Component Systems

Currently, JET has several officially supported component systems that implement PIE.

> - [Fozzie](https://vue.pie.design/) – a set of Vue components created by the legacy Just Eat web team
> - [Snacks](https://snacks.takeaway.com/) – a set of React components created by the legacy Takeaway design system team
> - [Skip PIE Project](https://react.pie.design/) – a set of React components created by the Skip web team

Reviewer: I feel like it doesn't matter too much who made them, rather what they are used for?
```suggestion
- [Fozzie](https://vue.pie.design/) – a set of Vue components for legacy Just Eat web applications
- [Snacks](https://snacks.takeaway.com/) – a set of React components for legacy Takeaway.com web applications
- [Skip PIE Project](https://react.pie.design/) – a set of React components for legacy SkipTheDishes web applications
```

Author: I think this one should stay as it is. Teams outside of Just Eat use the Vue components, and Snacks may be used outside of Takeaway.com platforms shortly (Scoober are talking about using some of them for instance).

We are also currently building the [PIE Web Component System](https://www.pie.design/storybook). The long-term aim will be to migrate JET teams over to this Web Component System, so we have one single source of truth for our global PIE components.

For more details on Fozzie, Snacks and Skip PIE, please check out the documentation portals linked above.

## PIE Project Structure

### The PIE Monorepo

As we look to consolidate our PIE components and tools, we wanted to ensure that all globally created components, tools and standards lived in a single place. That place is [the PIE Monorepo](https://github.com/justeattakeaway/pie).

{% notification {
type: "information",

> message: "For a detailed description of monorepo's and their pros and cons, checkout [this useful explainer on Wikipedia](https://en.wikipedia.org/wiki/Monorepo)."

Reviewer: ```suggestion
message: "For a detailed description of monorepos and their pros and cons, check out [this useful explainer on Wikipedia](https://en.wikipedia.org/wiki/Monorepo)."
```

} %}

The PIE Monorepo is currently organised into two distinct sections; apps and packages.

#### Apps – the `/apps` folder

> Projects that live in the `/apps` folder are self contained web applications that can be published and hosted on a server.

Reviewer: ```suggestion
Projects that live in the `/apps` folder are self-contained web applications that can be published and hosted on a server.
```
